src/ploigos_step_runner/utils/gpg.py

```diff
@@ -0,0 +1,36 @@
+"""Shared utils for dealing with gpg keys.
```

Comment:
@blakerollins can you also move the PodmanSign#__import_php_key function here, it should pretty much be a cut/paste of the funciton and the tests.